modules/apps/foundation/login/login-web/src/main/resources/META-INF/resources/login.jsp

```diff
@@ -154,18 +156,88 @@
 				<span id="<portlet:namespace />passwordCapsLockSpan" style="display: none;"><liferay-ui:message key="caps-lock-is-on" /></span>
 
 				<c:if test="<%= company.isAutoLogin() && !PropsValues.SESSION_DISABLED %>">
-					<aui:input checked="<%= rememberMe %>" name="rememberMe" type="checkbox" />
+					<aui:input checked="<%= rememberMe %>" inlineField="<%= true %>" name="rememberMe" type="checkbox" />
+				</c:if>
+
+				<%
+				String mvcRenderCommandName = ParamUtil.getString(request, "mvcRenderCommandName");
+
+				boolean showForgotPasswordIcon = false;
+
+				if (!mvcRenderCommandName.equals("/login/forgot_password") && (company.isSendPassword() || company.isSendPasswordResetLink())) {
+					showForgotPasswordIcon = true;
+				}
+				%>
+
+				<c:if test="<%= showForgotPasswordIcon %>">
+					<portlet:renderURL var="forgotPasswordURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
+						<portlet:param name="mvcRenderCommandName" value="/login/forgot_password" />
+					</portlet:renderURL>
+
+					<aui:a cssClass="pull-right" href="javascript:;" label="forgot-password" onClick='<%= renderResponse.getNamespace() + "showPopup('" + forgotPasswordURL + "', 'forgot-password');" %>'>?</aui:a>
 				</c:if>
 			</aui:fieldset>
 
 			<aui:button-row>
 				<aui:button cssClass="btn-lg" type="submit" value="sign-in" />
 			</aui:button-row>
-		</aui:form>
 
-		<liferay-util:include page="/navigation.jsp" servletContext="<%= application %>" />
+			<%
+			String mvcRenderCommandName = ParamUtil.getString(request, "mvcRenderCommandName");
+
+			boolean showCreateAccountLink = false;
+
+			if (!mvcRenderCommandName.equals("/login/create_account") && company.isStrangers() && !portletName.equals(PortletKeys.FAST_LOGIN)) {
+				showCreateAccountLink = true;
+			}
+			%>
+
+			<c:if test="<%= showCreateAccountLink %>">
+				<div class="separator"><!-- --></div>
+
+				<liferay-ui:message key="new-to-this-site" />
+
+				<portlet:renderURL var="createAccountURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
+					<portlet:param name="mvcRenderCommandName" value="/login/create_account" />
+				</portlet:renderURL>
+
+				<aui:a href="javascript:;" label="create-account" onClick='<%= renderResponse.getNamespace() + "showPopup('" + createAccountURL + "', 'create-account');" %>' />
+			</c:if>
+		</aui:form>
 
 		<aui:script sandbox="<%= true %>">
+			window.<portlet:namespace />showPopup = function(url, title) {
```

Comment:
This one also has the same issue as above with mixing the jQuery and AUI apis.
I would move this script into it's own `aui:script` block, and use a `Liferay.provide` to define the function.